pkgs/by-name/ps/pshash/package.nix

```
@@ -0,0 +1,41 @@
+{
+  haskellPackages,
+  fetchFromGitHub,
+  lib,
+}:
+haskellPackages.mkDerivation rec {
+  pname = "pshash";
+  version = "0.1.14.5";
+  src = fetchFromGitHub {
+    owner = "thornoar";
+    repo = "pshash";
+    rev = "v${version}";
+    hash = "sha256-CPvGoB8kavnLWSY131M5t8c12Mnji5wCVUVmcPOLwRo=";
```

```suggestion
    tag = "v${version}";
    hash = "sha256-b8cKdYExy88N6xFBT8duWpiXB+1M7bgGEVb2DKBqVUU=";
```
`tag` to avoid accidentally fetching a branch. And I got a different hash ^

---

That's weird, should be all good now, thanks for noticing